bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonBridgeHandler.java

```diff
@@ -58,6 +60,9 @@ public class InsteonBridgeHandler extends InsteonBaseThingHandler implements Bri
     private static final int RETRY_INTERVAL = 30; // seconds
     private static final int START_DELAY = 5; // seconds
 
+    private final ScheduledExecutorService insteonScheduler = ThreadPoolManager
+            .getScheduledPool(BINDING_ID + "-" + getThingId());
```

Comment:
Our convention is to use this built name: "OH-binding-" + thing.getUID()

Reply:
I was aware of the convention for naming `Thread` but not thread pools.

Looking at the existing calls in the codebase, none are following that convention. I don't have any problem using that convention but maybe we should update the existing thread pool names to do the same?

Reply:
Oups, you are right, I am mismatching thread name and thread pool name.